tests/jest.config.node.js

```diff
@@ -11,5 +11,4 @@ module.exports = {
     '/__tests__/',
     '/__node_tests__/',
   ],
-  testMatch: ['**/__node_tests__/**.js'],
```

Comment:
Should we move the existing file(s) to regular test files (or just remove them?)

Reply:
I'll think about that when we're getting closer to green tests :smile: